### ISSUE TITLE
It does not make sense to use ternary operator here

### ISSUE THREAD

**Issue by nkukarl** [peinanteng/leetcode#38]
https://github.com/peinanteng/leetcode/blob/fe45b660dc2f699611d5ecd46b5a30b0fcecbf0a/containerWithMostWater.py#L12

You should do something like
```
if height[left] < height[right]:
    left += 1
elif height[left] > height[right]:
    right -= 1
else:
    left += 1
    right -= 1
```
Then you don't need line 14.
Using ternary operator just makes the logic hard to follow.

**Comment by peinanteng** [comment]
I prefer to keep concise code style.
